### ISSUE TITLE
Add pyrm

### ISSUE THREAD

**Issue by avivajpeyi** [dmerejkowsky/pycp#28]
I really like pycp+pymv. 

Would it be possible to add a pyrm that performs an `rm`?

**Comment by dmerejkowsky** [comment]
Yup, but I'm not sure it belongs to this project. I tend to use `ncdu` to delete large repositories. You're welcome to try and work on a pull request, though.
